packages/api/resolvers/mutations/makeQuotationProposal.js

```diff
@@ -11,6 +11,7 @@ export default function (
   { userId },
 ) {
   log('mutation makeQuotationProposal', { quotationId, userId });
+  if (!quotationId) throw new Error('Invalid quotation ID provided');
```

Comment:
"When invalid or null ID is provided for a request" -> this means that error would catch mutation mutate(requiredParameter: null) or mutate(requiredParameter: "") calls?

Reply:
i added this explicit error because i want to distinguish between not found because item ID doesn't exist and not found because the user passed invalid ID to give more clear error message.